Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -686,7 +686,7 @@ namespace AzToolsFramework
                     if (changedLabel)
                     {
                         auto labelString = AZ::Dpe::Nodes::Label::Value.ExtractFromDomNode(valueAtSubPath).value_or("");
-                        changedLabel->SetText(QString::fromUtf8(labelString.data(), aznumeric_cast<int>(labelString.size())));
+                        changedLabel->setText(QString::fromUtf8(labelString.data(), aznumeric_cast<int>(labelString.size())));
```

Comment:
Just making sure you meant to change this back to `setText` after seeing your comment.

Reply:
both are good.